src/main/java/ui/listpanel/ListPanelCard.java

```diff
@@ -228,23 +236,57 @@ private void updateDetails() {
             issueDetails.getChildren().add(new Label(milestone.getTitle()));
         }
 
-        if (issue.getAssignee().isPresent() && guiElement.getAssignee().isPresent()) {
-            TurboUser assignee = guiElement.getAssignee().get();
-            Label assigneeNameLabel = new Label(issue.getAssignee().get());
-            assigneeNameLabel.getStyleClass().add("display-box-padding");
-
-            ImageView avatar = new ImageView();
-            if (assignee.getAvatarURL().length() != 0) {
-                Image image = assignee.getAvatar();
-                assert image != null;
-                avatar.setImage(image);
+        if (issue.isPullRequest()) {
+            HBox authorBox = createDisplayUserBox(guiElement.getAuthor(), issue.getCreator());
+            authorAssigneeBox.getChildren().add(authorBox);
+            if (issue.getAssignee().isPresent()) {
+                Label rightArrow = new Label(OCTICON_ARROW_RIGHT);
+                rightArrow.getStyleClass().addAll("octicon", "pull-request-assign-icon");
+                authorAssigneeBox.getChildren().add(rightArrow);
             }
+        }
+
+        if (issue.getAssignee().isPresent()) {
+            HBox assigneeBox = createDisplayUserBox(guiElement.getAssignee(), issue.getAssignee().get());
+            authorAssigneeBox.getChildren().add(assigneeBox);
+        }
+    }
+
+    /**
+     * Creates a box that displays a label of userName
+     * The avatar that belongs to the user will be prepended if TurboUser has it
+     * @param user
+     * @param userName
+     * @return
+     */
+    private HBox createDisplayUserBox(Optional<TurboUser> user, String userName) {
+        HBox userBox = setupUserBox();
+        Label authorNameLabel = new Label(userName);
+        addAvatarIfPresent(userBox, user);
+        userBox.getChildren().addAll(authorNameLabel);
+        return userBox;
+    }
+
+    private void addAvatarIfPresent(HBox authorBox, Optional<TurboUser> user) {
+        if (!user.isPresent()) return;
+        ImageView authorAvatar = getAuthorAvatar(user.get());
+        authorBox.getChildren().add(authorAvatar);
+    }
+
+    private HBox setupUserBox() {
+        HBox authorBox = new HBox();
+        authorBox.setAlignment(Pos.BASELINE_CENTER);
+        return authorBox;
+    }
 
-            HBox assigneeBox = new HBox();
-            assigneeBox.setAlignment(Pos.BASELINE_CENTER);
-            assigneeBox.getChildren().addAll(avatar, assigneeNameLabel);
-            issueDetails.getChildren().add(assigneeBox);
+    private ImageView getAuthorAvatar(TurboUser user) {
```

Comment:
Header comment for this method? I asked because the method seems to be doing more than a simple `get`. Also see if some functionality can be pushed to `TurboUser` class.